### DISCUSSION TITLE
"Infinite" slash loop in code search using regex

### DISCUSSION THREAD

**Discussion by Jayy001** [community/community#12988] [Other Feature Feedback, Questions, & Ideas] [answered]
If you type in `/(?i)^(thm{.*}|tryhackme{.*})$` (or any regex with / I believe) into https://cs.github.com/ it will keep recommending you to add slashes surrounding it

https://user-images.githubusercontent.com/56607897/158885962-c27c6141-3689-4b96-bfbf-9418aee040ce.mp4

Just thought it was quite interesting haha

**Comment by colinwm** [comment, marked as answer]
Good find! Thanks for the report. The slash should come after the $, not before... will fix.
